package.json

```diff
@@ -40,5 +40,10 @@
     "express": "~4.16.1",
     "http-errors": "~1.6.3",
     "morgan": "~1.9.1"
+  },
+  "devDependencies": {
+    "mocha": "^7.0.1",
+    "sinon": "^9.0.0",
```

Comment:
Is sinon just for mock data? Do we need it?

Reply:
we can remove this later